lib/ansible/config/base.yml

```diff
@@ -1348,6 +1348,13 @@ GALAXY_TOKEN:
   ini:
   - {key: token, section: galaxy}
   yaml: {key: galaxy.token}
+GALAXY_TOKEN_PATH:
```

Comment:
I'm planning on adding more options to this config file so this should be something like `GALAXY_CONFIG_PATH`. The extra config is to expand out the tokens so you can define extra servers and tokens for each individual server instead of just `token: <one token>`.